src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countyCourtJudgment/ContentProvider.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff.content.countyCourtJudgment;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS00120)